autoscale/src/main/resources/logback.xml

```diff
@@ -64,6 +64,10 @@
         <level value="${PERISCOPE_SQL_LOG_LEVEL:-INFO}"/>
     </logger>
 
+    <logger name="org.springframework.orm.jpa">
```

Comment:
Do you still need this? SRE already bugs us that we log too much and its too expensive.

Reply:
> I see check for the entitlement in periscope module in service layer, and in core module you are checking entitlement in flow and in syncer, but new API calls in autoscale API and the distrox cancel API can be called without entitlement. I would expect that we put all new logic behind an entitlement starting from API layer.
Will check about the entitlements on the API layer.

Reply:
Will take a look at this change. Guessing this may have been introduced to actually reduce JPA logging because it was far too noisy at the DEBUG level.